snapshotter/snapshotter.py

```diff
@@ -378,6 +394,13 @@ def snapshot(source, dest, debug=False, min_snapshots=3, extra_args=None):
     """
     date = _datetime()
     user, host, snapshots_root = _parse_path(dest)
+    if max_snapshots > -1 and max_snapshots - min_snapshots < 0:
```

Comment:
I was thinking about that but I think both ways are good, I can't see counter example at this point.